mlflow/store/artifact/databricks_artifact_repo.py

```diff
@@ -316,6 +316,24 @@ def _get_run_relative_artifact_path_for_upload(self, src_file_path, dst_artifact
             run_relative_artifact_path = self.run_relative_artifact_repo_root_path
         return run_relative_artifact_path
 
+    def _set_databricks_host_creds_to_credential_context(self):
+        db_creds = get_databricks_host_creds(self.databricks_profile_uri)
+        dbutils = _get_dbutils()
+        dbutils.endpoint.putMlflowProperties(
+            db_creds.host,
+            db_creds.ignore_tls_verification,
+            db_creds.token,
+            db_creds.username,
+            db_creds.password,
+        )
+
+    def _clear_credential_context(self):
+        dbutils = _get_dbutils()
+        dbutils.endpoint.clearMlflowProperties()
+
+    def _get_mlflowdbfs_path(self, artifact_path):
+        return "mlflowdbfs:///artifacts?run_id={}&path=/{}".format(self.run_id, artifact_path)
```

Comment:
Should we strip the leading "/" for the input `artifact_path` before using it? Could a user-error be a concern here? How does MLflowDBFS handle paths starting with "//" or "///"?

Reply:
mlflowdbfs can handle multiple slashes: https://github.com/databricks/runtime/blob/1ba6a8ee9e63226ecf6e5697b16ab43c34eb3831/sql/core/src/test/scala/com/databricks/mlflowdbfs/MlflowdbfsClientSuite.scala#L102-L103
However, I agree that we should check for leading slashes in `artifact_path` since that is not allowed by the definition of this parameter.